bin/resalloc

```diff
@@ -66,7 +76,7 @@ def main():
 
         conn = Connection(arg.connection)
         if 'ticket' == arg.subparser:
-            ticket = conn.newTicket(arg.tags)
+            ticket = conn.newTicket(arg.tags, sandbox=arg.sandbox)
```

Comment:
```suggestion
            ticket = conn.newTicket(tags=arg.tags, sandbox=arg.sandbox)
```

Reply:
I don't want to change calling convention if not necessary.

Reply:
From python point of view, it does not look good, but it is up on you. `newTicket` used named variables and it should work.

Reply:
I changed it so the call is consistent at least.   The `python3-resalloc` API would deserve some
orthogonal cleanup work, documentation, etc.  Let's postpone the work on this for now.